### DISCUSSION TITLE
Contradictory donation requests

### DISCUSSION THREAD

**Discussion by kitswas** [anuraghazra/github-readme-stats#2176] [Q&A] [answered]
### Describe the bug

Quoting the ReadMe.

> Love the project? Please consider [donating](https://www.paypal.me/anuraghazra) to help it improve!

> Are you considering supporting the project by donating? Please DO NOT!!

### Screenshots / Live demo link

![image](https://user-images.githubusercontent.com/90329875/194811955-004f84ab-9732-4ba1-97b5-898787e07b30.png)

https://github.com/anuraghazra/github-readme-stats/blob/master/readme.md

**Comment by rickstaa** [comment, marked as answer]
@kitswas This contradiction exists because, by default, although they are by no means required, donations to the GRS project are appreciated. 😄 The COVID donation notice was added at the onset of COVID-19 because of the alarming effects COVID had in India (see https://news.google.com/covid19/map?hl=en-US&mid=%2Fm%2F03rk0&gl=US&ceid=US%3Aen). Therefore, one is a temporary donation notice, and the other is the default donation notice.

**Reply by kitswas** [reply]
I had realized what had happened.

I meant to say the default notice should have been removed when the temporary one was brought in. 
The current situation only serves to confuse.

**Reply by rickstaa** [reply]
I see. I will leave this decision to @anuraghazra.

**Reply by rickstaa** [reply]
@kitswas I just closed this question by answering it. You can open an [issue](https://github.com/anuraghazra/github-readme-stats/issues) or [PR](https://github.com/anuraghazra/github-readme-stats/pulls) if you still want this to be fixed.
